CROSSCOMPILE.md

```diff
@@ -0,0 +1,91 @@
+# Crosscompilation for ARMHF
```

Comment:
Nice. We should do the same for aarch64. My raspberries are arm64 for example.

Reply:
good call, adding (but please do test before approving as i do not have access to a similar device yet)